src/main/java/minegame159/meteorclient/modules/combat/CrystalAura.java

```diff
@@ -422,11 +458,19 @@ public void onActivate() {
     @Override
     public void onDeactivate() {
         assert mc.player != null;
-        if (preSlot != -1) mc.player.inventory.selectedSlot = preSlot;
+        if (switchBack.get()) if (preSlot != -1) mc.player.inventory.selectedSlot = preSlot;
         for (RenderBlock renderBlock : renderBlocks) {
             renderBlockPool.free(renderBlock);
         }
         renderBlocks.clear();
+        if (target != null && resetRotations.get()) {
```

Comment:
What does this actually do? It seems to just set where the player is looking to where they are already looking.

Reply:
It's doing what the return mode used to do, reset your rotation server-side as you would still be looking at the support block/end crystal